fast64_internal/oot/README.md

```diff
@@ -23,7 +23,10 @@ Properties for empties/meshes can be found in the Blender object properties wind
 
 To export a scene, the "Scene Object" must be set in the "OOT Scene Exporter" section in the tool properties sidebar. When you click "Add Scene" this is set automatically.
 
-### Actors Variables
+### Actors
+To add actors to a scene, create a new Empty and parent it to a Room. The in the Object Properties panel select "Actor" as the Object Type. Use the "Select Actor ID" button to choose an actor, and then set the Actor Parameter value as desired (see the list of Actor Variables below). Finally, all actors you are using must be added to the parent Room's object dependecies, otherwise they will not load into the scene at all. To do this select the Room that your actor is parented to, select the "Objects" tab in its Object Properties window, and click "Add Item". Then "Search Object ID" to find the actor object you need. For example, if adding a Deku Baba actor (EN_DEKUBABA) you need to add the "Dekubaba" object to the Room's object dependencies.
```

Comment:
```suggestion
To add actors to a scene, create a new Empty and parent it to a Room. Then in the Object Properties panel select "Actor" as the Object Type. Use the "Select Actor ID" button to choose an actor, and then set the Actor Parameter value as desired (see the list of Actor Variables below). Finally, all actors you are using must be added to the parent Room's object dependencies, otherwise they will not load into the room at all. To do this select the Room that your actor is parented to, select the "Objects" tab in its Object Properties window, and click "Add Item". Then "Search Object ID" to find the actor object you need. For example, if adding a Deku Baba actor (EN_DEKUBABA) you need to add the "Dekubaba" object to the Room's object dependencies.
```

Reply:
Yikes, that's what I get for editing in a terminal without a spell check. thanks. I did not see the other PRs, those improvements are extremely helpful. great stuff. I'll have to test them out tomorrow.